calcom/cal.com
# feat: custom location for event types

## What does this PR do?

The feature allows the user to add custom location for his/her event types.

Fixes #2377

https://www.loom.com/share/d20b61dcf823401883a0ef411c3a332e

## Requirement/Documentation
Event types -> location of the event:

Custom Location: You can add your own text as location for booking.

## Type of change



## How should this be tested?
We can go to edit event and select custom location and then add your own typed location. The location will be shown on the ui. You can save the changes and refresh the page, changes will persist.
You can open your booking and can see the custom added location there as well.
You can also cancel and update that location.


## Mandatory Tasks


## Checklist


- I haven't added tests that prove my fix is effective or that my feature works
